lib/src/package.dart

```diff
@@ -140,6 +140,9 @@ class Package {
   /// This mechanism can be used to avoid loading pubspecs twice. It can also be
   /// used to override a pubspec in memory for trying out an alternative
   /// resolution.
+  ///
+  /// [referringPubspec] is the path to the pubspec that includes this one in
+  /// the workspace.
   factory Package.load(
     String dir,
```

Comment:
nit: documentation string refers to `packageDir`, but it's called `dir`. Probably the same, but would be neat to fix. (unrelated ofcourse no need to do that in this PR)

Reply:
Done